src/helpers/tabbable.js

```diff
@@ -18,6 +18,12 @@ function hidesContents(element) {
   // If the node is empty, this is good enough
   if (zeroSize && !element.innerHTML) return true;
 
+  // if the element is not of type Element e.g. shadowRoot
+  // we cannot go any further
+  if (!element.isPrototypeOf(Element)) {
+    return false;
+  }
+
```

Comment:
Maybe, we can wrap the `getComputedStyle` call on a try/catch, and, if it fails, return s `false`.

Reply:
That sounds like a more robust solution, I will make that change

Reply:
We need to be careful when using dom elements because they seem to differ when there is iframe on the modal.

See #871.